src/components/elements/livechat-link-text.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Localize } from 'components/localization'
+import { useLivechat } from 'components/hooks/use-livechat'
+import { LinkText } from 'components/elements'
+
+type LiveChatLinkTextProps = {
+    text?: string
```

Comment:
```suggestion
    text?: TString
```
please update this as well, sorry I forgot to add it on previous review. :pray: